core/client/src/main/java/alluxio/client/file/options/DeleteOptions.java

```diff
@@ -24,6 +25,7 @@
 @NotThreadSafe
 public final class DeleteOptions {
   private boolean mRecursive;
+  private boolean mRemoveUFSFile;
```

Comment:
And, should I add a option to user whether delete ufs's file and use alluixo's data persist it?